components/TProfile.vue

```diff
@@ -109,7 +334,7 @@
       </div>
     </div>
 
-    <div v-if="profile.type !== 'City'" class="flex space-x-2 p-4 justify-end">
+    <!-- <div v-if="profile.type !== 'City'" class="flex space-x-2 p-4 justify-end">
```

Comment:
You can comment code locally, when you develop, but please don't commit commented code